From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ricardo Pardini <ricardo@pardini.net>
Date: Sun, 20 Oct 2024 14:15:06 +0200
Subject: nanopi-r6c/r6s: otg usb_host0_xhci for ums

- enable u2phy0/u2phy0_otg and usb2phy0_grf;
- dr_mode otg usb_host0_xhci
---
 arch/arm/dts/rk3588s-nanopi-r6c-u-boot.dtsi | 17 ++++++++++
 arch/arm/dts/rk3588s-nanopi-r6s-u-boot.dtsi | 17 ++++++++++
 2 files changed, 34 insertions(+)

diff --git a/arch/arm/dts/rk3588s-nanopi-r6c-u-boot.dtsi b/arch/arm/dts/rk3588s-nanopi-r6c-u-boot.dtsi
index 111111111111..222222222222 100644
--- a/arch/arm/dts/rk3588s-nanopi-r6c-u-boot.dtsi
+++ b/arch/arm/dts/rk3588s-nanopi-r6c-u-boot.dtsi
@@ -1,3 +1,20 @@
 // SPDX-License-Identifier: (GPL-2.0+ OR MIT)
 
 #include "rk3588s-u-boot.dtsi"
+
+&u2phy0 {
+	status = "okay";
+};
+
+&usb2phy0_grf {
+	status = "okay";
+};
+
+&u2phy0_otg {
+	status = "okay";
+};
+
+&usb_host0_xhci {
+	dr_mode = "otg";
+	status = "okay";
+};
diff --git a/arch/arm/dts/rk3588s-nanopi-r6s-u-boot.dtsi b/arch/arm/dts/rk3588s-nanopi-r6s-u-boot.dtsi
index 111111111111..222222222222 100644
--- a/arch/arm/dts/rk3588s-nanopi-r6s-u-boot.dtsi
+++ b/arch/arm/dts/rk3588s-nanopi-r6s-u-boot.dtsi
@@ -1,3 +1,20 @@
 // SPDX-License-Identifier: (GPL-2.0+ OR MIT)
 
 #include "rk3588s-u-boot.dtsi"
+
+&u2phy0 {
+	status = "okay";
+};
+
+&usb2phy0_grf {
+	status = "okay";
+};
+
+&u2phy0_otg {
+	status = "okay";
+};
+
+&usb_host0_xhci {
+	dr_mode = "otg";
+	status = "okay";
+};
-- 
Armbian

